### ISSUE TITLE
Cannot import file with unicode path on Windows

### ISSUE THREAD

**Issue by Uflex** [alembic/alembic#206]
When trying to read an abc file using `AbcCoreFactory::IFactory::getArchive(const std::string & iFileName)`, the file fails to load if there is a unicode character in the path. This seems to be because the file name passed is a std::string and not a std::wstring. Have I missed something in the library or is this the kind of thing that was implied when writing that Windows support is experimental?

Error message in the console:

> HDF5-DIAG: Error detected in HDF5 (1.8.5) thread 0:
>   #000: C:\alembic\source\contrib\hdf5-1.8.5\src\H5F.c line 789 in H5Fis_hdf5(): unable to open file
>     major: Low-level I/O
>     minor: Unable to initialize object
>   #001: C:\alembic\source\contrib\hdf5-1.8.5\src\H5FD.c line 1088 in H5FD_open(): open failed
>     major: Virtual File Layer
>     minor: Unable to initialize object
>   #002: C:\alembic\source\contrib\hdf5-1.8.5\src\H5FDwindows.c line 360 in H5FD_windows_open(): unable to open file, errno = 2, error message = 'No such file or directory'
>     major: File accessability
>     minor: Unable to open file

Any help would be greatly appreciated.

**Comment by lamiller0** [comment]
Windows support isn't experimental.

Was the file written with a unicode name, or was it renamed?

**Comment by Uflex** [comment]
No, the file was written with ASCII name in an ASCII path. Moving the file into a folder containing extended Unicode characters seems to break the load (only when using HDF5 format).

**Comment by lamiller0** [comment]
Oh?  So when it's Ogawa it can load it correctly?

**Comment by Uflex** [comment]
Yes, it would appear so, although I was using several different files in both Ogawa and HDF5 format and it seemed that HDF5 was the common factor for not loading when placed in a folder with Unicode chars in the name. 

I need to try converting a file from Ogawa to HDF5 and check if the resulting file then has the issue when reimporting.

**Comment by lamiller0** [comment]
Were you able to confirm anything?

**Comment by lamiller0** [comment]
There are some hints for a possible Windows implementation here:
https://groups.google.com/forum/#!topic/alembic-discussion/vUMjv8LoNU4
